docs/en/operation/User-CLI.md

```diff
@@ -1152,6 +1152,12 @@ Here are the additional properties possible for the `-o` options:
   * `catalog.db.sync.threads`: number of parallel threads to use to sync with the UDB. If too large,
   the sync may overload the UDB, and if set too low, syncing a database with many tables make take
   a long time. The default is `4`.
+  * `udb-glue.aws.<MOUNT_PROPERTY>`: specify the mount option for the Glue udb, the mount options
```

Comment:
maybe we should have 2 sub-sections, one for the hive udb type, and one for the glue udb type. Then, in each section, we can talk about the specific options. I don't think hive has any hive specific options, so the glue sub-section would discuss the glue-specific ones.

Reply:
I have divided the doc into two part with Hive and Glue, please check it.